indico/web/client/js/jquery/extensions/global.js

```diff
@@ -247,19 +247,31 @@ $(document).ready(function() {
   showFormErrors();
 
   // Show form creation dialog if hash is present in the URL
-  var match = location.hash.match(/^#create-event:(lecture|meeting|conference)(?::(\d+))?$/);
+  var match = location.hash.match(/^#create-event:(lecture|meeting|conference)(?::(\d*))(?::(\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b))?$/);
```

Comment:
Please, no uuid regex :D I'd go for something like this...

    const match = location.hash.match(/^#create-event:(lecture|meeting|conference)(?::(\d*)(?::([a-f0-9-]{36}))?)?$/);

Like this it will still match all these cases, but without this awfully long regex:

- `#create-event:meeting`
- `#create-event:meeting:123`
- `#create-event:meeting::c730a998-c8eb-4d85-b398-c8821354213b`
- `#create-event:meeting:123:c730a998-c8eb-4d85-b398-c8821354213b`

`match[2]` is always the category id (empty string or undefined if missing)
`match[3]` is the uuid (or undefined if missing)

Reply:
Sure, updated :)